check_yum.py

```diff
@@ -322,33 +322,25 @@ def get_all_updates(self):
         if self.verbosity >= 4:
             for section in output2:
                 print("\nSection:\n%s\n" % section)
-        if len(output2) > 2 or \
+        if len(output2) > 2 or (not self.no_cache_update and \
```

Comment:
If `no_cache_update` is set then there will be no "Setting up repositories" (or any other output, just the plain updates)